app/views/katello/api/v2/hosts/base.json.rabl

```diff
@@ -1,7 +1,31 @@
 object @resource
 
 @resource ||= @object
+@facet = @resource.content_facet
 
 attributes :id, :name, :description
-attributes :content_view, :content_view_id
-attributes :lifecycle_environment => :environment
+if @facet
+  node :content_view do
+    if @facet.single_content_view_environment?
+      content_view = @facet.single_content_view
+      {
+        :id => content_view.id,
+        :name => content_view.name,
+        :composite => content_view.composite?
+      }
+    else
+      "multiple"
```

Comment:
If we don't show the multiple LCEs on the UI, do we need to show a "multiple" placeholder on the API? Showing the same CV Environment that's on the UI would be more consistent.